bundles/org.openhab.binding.airgradient/src/main/java/org/openhab/binding/airgradient/internal/model/Measure.java

```diff
@@ -0,0 +1,169 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.airgradient.internal.model;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * Data model class for a single measurement from AirGradients API.
+ *
+ * @author Jørgen Austvik - Initial contribution
+ */
+@NonNullByDefault
+public class Measure {
+
+    /**
+     * Returns a location id that is guaranteed to not be null.
+     * 
+     * @return A non null location id.
+     */
+    public String getLocationId() {
+        String loc = locationId;
+        if (loc != null) {
+            return loc;
+        }
+
+        return "";
+    }
+
+    /**
+     * Returns a location name that is guaranteed to not be null.
+     *
+     * @return A non null location name.
+     */
+    public String getLocationName() {
+        String name = locationName;
+        if (name != null) {
+            return name;
+        }
+
+        return "";
```

Comment:
```suggestion
        retun (name != null) ? name : "";
```
Not wrong, but simpler to write. Same for other occurences in this class.

Reply:
Thanks!